### ISSUE TITLE
Failing docker image on master - firmwares not found

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-user-env#35]
For some reasons downloaded firmwares end up in root directory instead of the proper one. 

```
$ docker run  -it registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env:d95db635f7d9af0c1ae70384cdffda899b6498b0 /bin/sh

Unable to find image 'registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env:d95db635f7d9af0c1ae70384cdffda899b6498b0' locally
d95db635f7d9af0c1ae70384cdffda899b6498b0: Pulling from satoshilabs/trezor/trezor-user-env/trezor-user-env
Digest: sha256:3419cd65f79b83918618713d5e19632065e00914d5dd2e66f68deb3b4ac1ad48
Status: Downloaded newer image for registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env:d95db635f7d9af0c1ae70384cdffda899b6498b0

114559d32878:/trezor-user-env# ls
README.md                    trezor-emu-core-v2.0.8       trezor-emu-core-v2.2.0       trezor-emu-legacy-v1.7.2
controller                   trezor-emu-core-v2.0.9       trezor-emu-core-v2.3.0       trezor-emu-legacy-v1.7.3
docker                       trezor-emu-core-v2.1.0       trezor-emu-core-v2.3.1       trezor-emu-legacy-v1.8.0
docs                         trezor-emu-core-v2.1.1       trezor-emu-core-v2.3.2       trezor-emu-legacy-v1.8.1
firmware                     trezor-emu-core-v2.1.2       trezor-emu-core-v2.3.3       trezor-emu-legacy-v1.8.2
init.sh                      trezor-emu-core-v2.1.3       trezor-emu-core-v2.3.4       trezor-emu-legacy-v1.8.3
run_e2e_tests.sh             trezor-emu-core-v2.1.4       trezor-emu-legacy-v1-master  trezor-emu-legacy-v1.9.0
shell.nix                    trezor-emu-core-v2.1.5       trezor-emu-legacy-v1.6.2     trezor-emu-legacy-v1.9.1
suite                        trezor-emu-core-v2.1.6       trezor-emu-legacy-v1.6.3     trezor-emu-legacy-v1.9.2
trezor-emu-core-v2-master    trezor-emu-core-v2.1.7       trezor-emu-legacy-v1.7.0     trezor-emu-legacy-v1.9.3
trezor-emu-core-v2.0.10      trezor-emu-core-v2.1.8       trezor-emu-legacy-v1.7.1     trezord-go
```

Probably introduced by #33.

**Comment by darwin** [comment]
My guess: https://github.com/trezor/trezor-user-env/blob/d95db635f7d9af0c1ae70384cdffda899b6498b0/docker/Dockerfile#L30

We are the script expects to be run by bash, not sh. I think nix-shell should run the script as executable so the shebang is respected.

**Comment by tsusanka** [comment]
I thought so too but I am just confused why this works just fine on my machine.. I will try to remove the `/bin/sh` as you suggest though.

**Comment by tsusanka** [comment]
That seems to do the trick, thank you for suggestion! Fixed via f78ba6ef20f67cf5950650ec0e071d740a795834.
